tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Utility/ConfigSettings/SearchSettings.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.Legal.MatterCenter.Utility
     public class SearchSettings
     {
         public string SearchResultSourceID { get; set; }
-
+        public string ManagedPropertyExtension { get; set; }

```

Comment:
Added a new key called ManagedPropertyExtension which will have value called MC for matter center